CMSgov/easi-app
# [NOREF] Fix possible nil dereference when calling LCID Email code

# NORE

## Changes and Description

- Adjusts parameters to email methods that deal with `lifecycleCostBaseline` to modify them to be pointers so they're handled appropriately when they're not passed.

## How to test this change

- `scripts/dev up`
- Create a system intake
- Use Postman to take the `createSystemIntakeActionIssueLCID` action and _DO NOT_ pass a `costBaseline`.
- Ensure the email looks good and the action is still appropriately taken

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
